### PR TITLE
Bl vt estimate next purchase

### PR DESCRIPTION
**Pull request by veeteeran**
Hi all. I made a mistake while working with the merge conflict. I think it was in the return statement starting on line 136. Missing parenthesis perhaps? The app is compiling and running on my machine. The filtering looks good and so does the estimate next purchase date. I've pushed to the BL-VT branch in case something else is funny. I haven't reverted the main branch. Lesson learned. Don't try to fix a merge conflict before going to bed. 😦  😴

### CODE REVIEW

File: src/components/List.js [hunk 1]
```diff
@@ -101,42 +92,22 @@ const List = ({ token }) => {
...
-                ?.toLowerCase()
-                ?.includes(searchInput.toLowerCase().trim()),
-            )
-            ?.map((doc) => (
+          docs.map((doc) => {
+            return (
               <li key={doc.id}>
                 <input
                   type="checkbox"
-                  aria-label="purchased-checkbox"
```
Comment (veeteeran) [inline src/components/List.js:139]: I noticed I don't have this line. Should we put it back in? Looks like it will now be on line 140.

Comment (rachelrohrbach) [reply src/components/List.js:139]: I added this for accessibility, so yes if we could put it back in.

File: src/components/List.js [hunk 2]
```diff
@@ -132,29 +132,29 @@ const List = ({ token }) => {
...
-                  <p>You haven't purchased {doc.itemName} yet.</p>
-                )}
-              </li>
-            );
-          })}
+            ?.map((doc) => {
+              return (
+                <li key={doc.id}>
+                  <input
+                    type="checkbox"
```
Comment (laurenmbeatty) [inline src/components/List.js:139]:
```suggestion
                    type="checkbox"
                    aria-label="purchased-checkbox"
```

**Review by laurenmbeatty — APPROVED** [review]
Yep! Looks good 👍 ! You can just commit the aria-label line right here in gh.

**Pull request merged**